exonum-java-binding-core/rust/integration_tests/tests/node_context.rs

```diff
@@ -31,95 +30,76 @@ lazy_static! {
 }
 
 #[test]
-fn submit_valid_transaction() {
-    let jclass = JObject::null().into();
-    let (mut node, app_rx) = create_node();
-    let node_handle_guard = as_handle(&mut node);
-    let node_handle = node_handle_guard.get();
-    let (java_transaction, _raw_message) = create_mock_transaction(&EXECUTOR, true);
-    let marker_raw = RawMessage::from_vec(vec![1, 2, 3]);
-    let raw_message = marker_raw.clone();
-    unwrap_jni(EXECUTOR.with_attached(move |env: &JNIEnv| {
-        let submit = || {
-            let message = message_from_raw(env, &raw_message)?;
-            Java_com_exonum_binding_service_NodeProxy_nativeSubmit(
-                env.clone(),
-                jclass,
-                node_handle,
-                *java_transaction.as_obj(),
-                *message,
-                0,
-                raw_message.len() as i32,
-            );
-            let exception: JObject = env.exception_occurred()?.into();
-            assert!(exception.is_null());
-            Ok(())
-        };
-        unwrap_jni_verbose(&env, submit());
-        Ok(())
-    }));
+fn submit_transaction() {
```

Comment:
Please make this test clearer: currently the code under test is mixed with the setup code and it is hard to distinguish the two.